modules/data.atmosphere/R/download.GFDL.R

```diff
@@ -142,10 +144,21 @@ download.GFDL <- function(outfolder, start_date, end_date, site_id, lat.in, lon.
         start_url, "00-", end_url, "23.nc"
       )
       dap_file <- paste0(dap_base, dap_end)
-      dap <- ncdf4::nc_open(dap_file)
+      dap <- ncdf4::nc_open(dap_file, suppress_dimvals = TRUE)
+
+      # Sanity check:
+      # We're saving the data with timestamps at the end of the interval,
+      # while GFDL-supplied timestamps vary slightly -- some vars are
+      # timestamped in middle of interval, others at end.
+      # But if these disagree by more than 3 hours, we have a problem.
+      raw_time <- ncdf4::ncvar_get(dap, "time", start = time_offset, count = obs_per_year)
+      converted_time <- udunits2::ud.convert(raw_time, dap$dim$time$units, dim$time$units)
```

Comment:
I _think_ this is fine here, but you want to be _very_ careful using `ud.convert` on times. `ud.convert` is not designed for calendar conversions, but rather for _physical_ conversions, so it knows, for example, that there are not exactly 365 days in a year:

```
ud.convert(1, "year", "days")
# [1] 365.2422
```

We may want to invest some time into our own wrapper around `lubridate` and other utilities that know how to take into account some of these met-specific quirks (e.g. 360 day years, no leap years, etc.).

Reply:
This is a good point, but also one I'm sore about: the the udunits documentation [specifically discourages](https://www.unidata.ucar.edu/software/udunits/udunits-current/doc/udunits/udunits2lib.html#Time) using it for time, but the netCDF time units were [literally designed to be converted by udunits](https://www.unidata.ucar.edu/software/netcdf/time/recs.html) and the CF conventions [incorporate the udunits definitions](http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch04s04.html) but then recommend "caution" with units like year and month that vary in length. Bottom line: a better wrapper would be great, but I'm not sure it can save us from an overflexible standard.